### ISSUE TITLE
Channels symbolics

### ISSUE THREAD

**Issue by proofit404** [django/channels#591]
We have Django official logo. Since channels is an official django project I maybe reasonable to have channels channels logo.

I want to print stickers and t-shirt with it after all.

Maybe [this](https://heroku-blog-files.s3.amazonaws.com/posts/1473343843-django-channels-header.png) can be a good start.

**Comment by andrewgodwin** [comment]
Haha, I think Heroku own the copyright on that one. I'm not particularly artistically inclined, I'll see if I can rustle one up though.

**Comment by nicokant** [comment]
@proofit404 @andrewgodwin I'd like to help with this issue! I took inspiration from the Heroku blog post you suggested. It's still a prototype, so let me know what you think!

![channels-logo](https://user-images.githubusercontent.com/6927678/36393573-789e7bc2-15b0-11e8-9a31-49f58e03a5c9.png)

**Comment by andrewgodwin** [comment]
I quite like it, but I generally prefer logos to have their text and their graphic element separate! Could you try a version with the text off to the right? And maybe make the lines and elements thicker/larger so they're more visible at small sizes? I'm just trying to think of how it would look in small-square, small-rectangle and similar forms.

**Comment by proofit404** [comment]
@nicokant Thanks a lot for the effort! Andrews comment seems reasonable.

**Comment by nicokant** [comment]
Yeah, I think you are right. I wasn't really sure about keeping together graphic elements and text. In a few days, I hope to have something ready to show you!

**Comment by maxg203** [comment]
@nicokant Did you end up doing another version? I think Django Channels needs a logo too!

**Comment by Rajesh-Yogeshwar** [comment]
Not to undermine anybody's efforts. But just few more options ;)

![dc1](https://user-images.githubusercontent.com/7311146/39110234-fcbe3afa-46ed-11e8-88f8-ffe7e88906dd.png)
![dc2](https://user-images.githubusercontent.com/7311146/39110235-fce81f5a-46ed-11e8-9e7f-7ab28a775310.png)

**Comment by kradem** [comment]
Not to undermine anybody's efforts too, of course, but this is an example suggesting we could stick to the roots both in the means of colors and font and in minimalist nature of the original logo.

![django-channels-logo](https://user-images.githubusercontent.com/7360633/42305551-b9dcc220-802b-11e8-9ecc-f842cf52f8ba.png)

**Comment by KakarN** [comment]
Another "Not to undermine anybody's efforts." too 😄
![Uploading django-channels-logo-1.jpg…]()
![Uploading django-channels-logo-2.jpg…]()

**Comment by rajeshyogeshwar** [comment]
The concept for C is really good, upvoted.

**Comment by lorentzen** [comment]
Hi, I have made this design: https://www.doprax.com/community/proposal-for-official-django-channels-logo/

![Django-Channels-logo-neg](https://github.com/django/channels/assets/3798895/bac521c0-705a-4d0e-a2a6-38da581a02b3)
![Django-Channels-mark](https://github.com/django/channels/assets/3798895/cd1240f4-40ab-422c-9d5b-117d0b71eadc)
<img width="1028" alt="Django-Channels-icon-round@2x" src="https://github.com/django/channels/assets/3798895/dfb78e3e-8280-4f7d-963f-c3bffe68d25b">

I hope you like it! Please comment

**Comment by lorentzen** [comment]
![Django-Channels-logo-small-pos](https://github.com/django/channels/assets/3798895/d83d3378-c6a0-4e7b-8bec-76f3731c80d4)
